test/e2e/e2e-tests.yaml

```diff
@@ -103,6 +103,10 @@ data:
         value: "0"
       - name: AZURE_CLOUD
         value: "AZURE_PUBLIC_CLOUD"
+      - name: IS_CONTAINERLOGV2
```

Comment:
as we discussed, lets update the azuremonitor-containers helm chart to set environment variable  AZMON_KUBERNETES_METADATA_ENABLED and with value true when IS_TEST is true.

Reply:
changed